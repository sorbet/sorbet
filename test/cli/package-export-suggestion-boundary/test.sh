cd test/cli/package-export-suggestion-boundary || exit 1

../../../main/sorbet --silence-dev-message --stripe-packages --max-threads=0 . 2>&1
