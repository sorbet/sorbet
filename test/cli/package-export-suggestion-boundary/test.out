example/__package.rb:4: Unable to resolve constant `A` https://srb.help/5002
     4 |  export Some::Example::A
                 ^^^^^^^^^^^^^^^^
  Did you mean `Some::Example::B`? Use `-a` to autocorrect
    example/__package.rb:4: Replace with `Some::Example::B`
     4 |  export Some::Example::A
                 ^^^^^^^^^^^^^^^^
    example/b.rb:4: `Some::Example::B` defined here
     4 |  class B
          ^^^^^^^

example/__package.rb:5: Unable to resolve constant `A` https://srb.help/5002
     5 |  export Test::Some::Example::A
                 ^^^^^^^^^^^^^^^^^^^^^^
  Did you mean `Test::Some::Example::B`? Use `-a` to autocorrect
    example/__package.rb:5: Replace with `Test::Some::Example::B`
     5 |  export Test::Some::Example::A
                 ^^^^^^^^^^^^^^^^^^^^^^
    example/test/b.rb:4: `Test::Some::Example::B` defined here
     4 |  class B
          ^^^^^^^

example/__package.rb:6: Unable to resolve constant `Exmpl` https://srb.help/5002
     6 |  export Some::Exmpl
                 ^^^^^^^^^^^
  Did you mean `Some::Example`? Use `-a` to autocorrect
    example/__package.rb:6: Replace with `Some::Example`
     6 |  export Some::Exmpl
                 ^^^^^^^^^^^
    example/b.rb:3: `Some::Example` defined here
     3 |module Some::Example
        ^^^^^^^^^^^^^^^^^^^^

example/__package.rb:7: Unable to resolve constant `So` https://srb.help/5002
     7 |  export So
                 ^^
Errors: 4
